.github/workflows/conformance.yml

```diff
@@ -151,7 +151,7 @@ jobs:
         working-directory: ./tests
 
       - name: Upload profile to release
-        if: ${{ inputs.k8s-version == 'latest' && startsWith(github.ref, 'refs/tags/') }}
```

Comment:
Didn't we add this check so we don't try and upload the profile for every conformance test run?

Reply:
No, the `--clobber` flag ensures that the last one wins. We run on latest for multiple runs anyway.